Jenkinsfile

```diff
@@ -41,6 +42,11 @@ pipeline {
             defaultValue: false,
             description: 'Whether to also push distribution ZIP archive to S3.',
         )
+        booleanParam(
+                name: 'PUSH_TO_GOOGLE_CLOUD',
```

Comment:
nit: for naming consistency, probably `push_to_vorvan` makes more sense... although since this is public repo, I'd also say putting to the description that its going to `h2o.ai maintained gcr.io repo vorvan` would also be ok.